docs/src/components/CopyToClipboardButton.tsx

```diff
@@ -0,0 +1,74 @@
+import Tooltip, { TooltipProps } from '@mui/material/Tooltip';
+import React, { useState } from 'react';
+import { copyTextToClipboard } from '../shared';
+import { isMobile } from 'react-device-detect';
+import CopyAllIcon from '@mui/icons-material/CopyAll';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+
+type Position = 'bottom' | 'top' | 'left' | 'right';
+type CopyToClipboardProps = Omit<
+    Omit<Omit<Omit<Omit<Omit<TooltipProps, 'placement'>, 'open'>, 'children'>, 'onMouseEnter'>, 'onMouseLeave'>,
+    'title'
+> & {
+    duration?: number;
+    position?: Position;
+    title?: string;
+    copiedPosition?: Position;
+    copiedTitle?: string;
+    copyText: string;
+};
+export const CopyToClipboard: React.FC<CopyToClipboardProps> = (props) => {
+    const {
+        duration = 1000,
+        position = 'bottom',
+        title = '',
+        copiedPosition = 'bottom',
```

Comment:
We would never want to have a tooltip appear at two different positions in one component. `position` and `copiedPosition` is redundant.

Reply:
I removed copiedPosition. We will be just keeping position prop, if user wants to change the position of tooltip